packages/sysutils/fuse/package.mk

```diff
@@ -21,8 +21,8 @@ PKG_VERSION="2.9.4"
 PKG_REV="1"
 PKG_ARCH="any"
 PKG_LICENSE="GPL"
-PKG_SITE="http://sourceforge.net/projects/fuse/"
-PKG_URL="$SOURCEFORGE_SRC/fuse/fuse-2.X/$PKG_VERSION/$PKG_NAME-$PKG_VERSION.tar.gz"
+PKG_SITE="https://github.com/libfuse/libfuse/"
+PKG_URL="https://github.com/libfuse/libfuse/releases/download/fuse_2_9_4/fuse-2.9.4.tar.gz"
```

Comment:
Untested, and might not work in `sh`, what about:

`PKG_URL="https://github.com/libfuse/libfuse/releases/download/${PKG_NAME}_${PKG_VERSION//./_}/$PKG_NAME-$PKG_VERSION.tar.gz"`?